functional/platform/nspawn.go

```diff
@@ -302,14 +302,18 @@ func (nc *nspawnCluster) createMember(id string) (m Member, err error) {
 
 		// minimum requirements for running systemd/coreos in a container
 		fmt.Sprintf("mkdir -p %s/usr", fsdir),
+		//fmt.Sprintf("mkdir -p %s/lib", fsdir), //ubuntu
```

Comment:
We definitely shouldn't leave commented lines in here. Are the lines totally superfluous now? The tests all work fine without?